README.md

```diff
@@ -23,3 +23,7 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+V is the number of vertices in a graph. The while loop runs V times as each node is dequeued. In the main loop for each node you check all its neighbors which could go over all the edges E. Each enqueue operation is O(V) and then there is E edges total so the final complexity is O(V*E).
```

Comment:
But you're doing enqueues for each node, right? Shouldn't there be a $|V|^2$ somewhere then?